zgrab_schema.py

```diff
@@ -202,7 +202,8 @@
     }),
     "session_ticket":SubRecord({
         "value":Binary(),
-        "length":Integer()
+        "length":Integer(),
```

Comment:
If it's important (e.g. some sort of security parameter), I'm fine storing length. It sounds like in this case it is. @aaspring, what's the length range, and do you think it makes more sense to give length in bits or bytes?